bin/iam-token.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# the iam-token.sh scripts generate new IAM token
+# it relies on the following environment variables
+# IAM_CLIENT_ID client id value
```

Comment:
Can you please clarify whether these variables are supposed to contain the final value? Or are they suppose to contain a file path which will then contain the value? I guess it's the latter from the code, but we better make it explicit here.

Reply:
I added it to the comments, these are values which we obtain from IAM provider, i.e. it is not file names.